src/windows/process.rs

```diff
@@ -240,18 +239,22 @@ impl CPUsageCalculationValues {
         }
     }
 }
-static WINDOWS_8_1_OR_NEWER: Lazy<bool> = Lazy::new(|| unsafe {
-    let mut version_info: OSVERSIONINFOEXW = MaybeUninit::zeroed().assume_init();
+static WINDOWS_8_1_OR_NEWER: OnceLock<bool> = OnceLock::new();
```

Comment:
Same: if it's only used in the `windows_8_1_or_newer` function, please move it inside it.